src/sgame/sg_spawn.cpp

```diff
@@ -732,6 +734,9 @@ static void G_ParseField( const char *key, const char *rawString, gentity_t *ent
 
 			( ( float * ) entityDataField ) [ 0 ] = tmpFloatData[ 0 ];
 			( ( float * ) entityDataField ) [ 1 ] = tmpFloatData[ 1 ];
+			if ( !Q_stricmp( key, "angles" ) && entity->preserveRoll ) { // HACK: "roll" keyword must overwrite "angles"
+				break;                                                   // Proper fix requires rewriting entity parsing
+			}
```

Comment:
I prefer if you format the comment this way:

```c++
			// HACK: "roll" keyword must overwrite "angles"
			// Proper fix requires rewriting entity parsing
			if ( !Q_stricmp( key, "angles" ) && entity->preserveRoll ) {
				break;                                                   
			}
```

Reply:
I've removed that hack altogether now in favour of a different approach.